control_plane/attachment_service/src/service.rs

```diff
@@ -239,15 +241,19 @@ impl Service {
         let mut compute_notifications = Vec::new();
 
         // Populate intent and observed states for all tenants, based on reported state on pageservers
+        let nodes_online = self.initial_heartbeat_round().await;
```

Comment:
Maybe pull this up one line, so that the "Populate intent..." comment still sits next to the block doing that?

Reply:
https://github.com/neondatabase/neon/pull/7092/commits/e603c0c3489ff462f41f0d6c64fc1a5a42ee9394